components/reports/reports-filters.tsx

```diff
@@ -223,25 +224,27 @@ export const FilterItems: React.FC<FilterItemsProps> = ({
 			{isMobileFilter && (
 				<DrawerFooter className="flex flex-wrap justify-center gap-2">
 					<section className="flex flex-wrap gap-2">
-						<DrawerClose className="flex gap-2 w-full">
-							<Button
-								variant={"secondary"}
-								className="flex-1"
+						<div className="flex gap-2 w-full">
+							<DrawerClose
+								className={cn(
+									buttonVariants({ variant: "secondary" }),
+									"flex-1",
+								)}
 								onClick={() =>
 									updateSearchParams(filters.filter(([key, _]) => key === "q"))
 								}
 							>
 								Clear filters
-							</Button>
-							<Button className="flex-1" variant={"outline"}>
+							</DrawerClose>
+							<DrawerClose
+								className={cn(buttonVariants({ variant: "outline" }), "flex-1")}
+							>
 								Cancel
-							</Button>
-						</DrawerClose>
+							</DrawerClose>
+						</div>
 					</section>
-					<DrawerClose>
-						<Button onClick={applyFilters} className="w-full">
-							Apply
-						</Button>
+					<DrawerClose onClick={applyFilters} className="w-full">
+						Apply
 					</DrawerClose>
```

Comment:
was blue button bg removed intentionally? imo was better with it.

current
<img width="319" alt="Screenshot 2024-04-05 at 4 42 48 PM" src="https://github.com/VoiceDeck/app/assets/79813968/62bdb67a-1215-4760-b4f9-827bd27eb4ab">

previous
<img width="319" alt="Screenshot 2024-04-05 at 4 43 22 PM" src="https://github.com/VoiceDeck/app/assets/79813968/639d2dc1-3061-4eb8-bad5-a756971281c2">

Reply:
that was not intentional hmm, nice catch!